packages/jupyterlab-gridstack/src/editor/components/notebookButtons.ts

```diff
@@ -37,7 +37,11 @@ export class EditorButton
       onClick: () => {
         this._commands.execute('docmanager:open', {
           path: panel.context.path,
-          factory: 'Voila GridStack'
+          factory: 'Voila GridStack',
+          options: {
+            mode: 'split-right',
+            ref: panel.id
+          }
```

Comment:
In that use case, should we connect the notebook panel disposal with the editor disposal as for cloned output view.

Reply:
Sorry, I'm not following you. What do you mean by connecting the notebook panel disposal with the editor disposal?

Reply:
Okay, do you mean binding the editor to the notebook so when closing the notebook will close the editor but just in the case when the editor it's opened from the notebook?

Reply:
Yes exactly - only in the case the editor has been opened from the notebook toolbar button. The gridstack editor will be closed when the notebook is closed.

Reply:
Nice, I like the idea.

cc @jtpio thoughts?

Reply:
Maybe it could be tried out yes.

Thinking that for some it might be confusing if the editor is closed automatically when the notebook is closed, while the editor can be open as a standalone widget and would have to be closed explicitly (two behaviors for the same widget).

Reply:
That's true.
Anyway, as it's already implemented let's try it and see if someone complains.